### PR TITLE
Creating tool to add data to new INFO field

### PR DESCRIPTION
**Pull request by chrisamiller**

### CODE REVIEW

File: vcf_annotation_tools/vcf_info_annotator.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    if args.source:
+        od['Source'] = args.source
+    if args.version:
+        od['Version'] = args.version
+    new_header.add_info_line(od)
+    output_file = args.output_vcf
+    return vcfpy.Writer.from_path(output_file, new_header)
+
+def define_parser():
+    parser = argparse.ArgumentParser("vcf-generic-annotator")
```
Comment (susannasiebert) [inline vcf_annotation_tools/vcf_info_annotator.py]: What do you want the entry point to be named? Here it's `vcf-generic-annotator` but the file and class is VcfInfoAnnotator. This should be consistent.

**Review by susannasiebert — CHANGES_REQUESTED** [review]
If you want to make this accessible on the command line you will need to add an entry to the entry_point->console_scripts array in setup.py.

**Comment by chrisamiller** [comment]
Just pushed fixes to both of those issues

**Comment by chrisamiller** [comment]
Pinging @susannasiebert before merging

**Review by susannasiebert — APPROVED** [review]

**Comment by susannasiebert** [comment]
Looks good to me. The only other thing that would be great is to add a test case to check that the annotation actually works.

**Review by susannasiebert — APPROVED** [review]
+1 once tests pass. Thanks for this addition to the package.

**Pull request merged**